### PR TITLE
[now-cli] Disable `--prod` and `--target` for Now 1.0 deployments

### PR DESCRIPTION
**Pull request by lucleray**
This PR disables Now 1.0 production deployments with the following error message:

> Option --prod is not supported for Now 1.0 deployments. To manually alias a deployment, use `now alias` instead.

It looks like this:
Capture d’écran 2019-10-23 à 19 42 09

Also disables `--target` for Now 1.0 deployments.

### CODE REVIEW

File: packages/now-cli/src/commands/deploy/legacy.ts [hunk 1]
```diff
@@ -304,6 +304,16 @@ export default async function main(
     `You are using an old version of the Now Platform. More: ${link(infoUrl)}`
   );
 
+  if (argv.prod) {
+    error(
+      `Now 1.0 does not support Production Deployments. Please use Now 2.0 to take advantage this feature: ${link(
+        'https://zeit.co/upgrade'
```
Comment (TooTallNate) [inline packages/now-cli/src/commands/deploy/legacy.ts]: Should we also tell the user how to achieve what they want for the 1.0 deployment? (i.e. manual `now alias` or whatever?)

Comment (leo) [reply packages/now-cli/src/commands/deploy/legacy.ts]: Let's do that 👍

Comment (lucleray) [reply packages/now-cli/src/commands/deploy/legacy.ts]: > Option --prod is not supported for Now 1.0 deployments. To manually alias a deployment, use now alias instead.

(and I drop the "update" message)
What do you think @leo @TooTallNate ?

Comment (leo) [reply packages/now-cli/src/commands/deploy/legacy.ts]: That's great! But let's make `now alias` use a code snippet.

**Review by TooTallNate — APPROVED** [review]

**Pull request merged**